src/callbacks/onResize/addYAxisTicks.js

```diff
@@ -13,17 +14,23 @@ export default function addYAxisTicks() {
 
         for (let i = 0; i < probs.length; i++) {
             probs[i].quantile = quantile(
-                this.measure_data.map(d => +d[this.config.y.column]).sort(),
+                this.measure_data.map(d => +d[this.config.y.column]).sort((a, b) => a - b),
                 probs[i].probability
             );
         }
 
         const ticks = [probs[1].quantile, probs[3].quantile];
```

Comment:
I'm wondering if the log ticks should be treated differently - they might get lil squished

Reply:
What's a log equivalent of quantiles?

Reply:
Created #76 for this